packages/react-sdk/src/components/span.ws.tsx

```diff
@@ -0,0 +1,15 @@
+import { BrushIcon } from "@webstudio-is/icons";
+import type { WsComponentMeta } from "./component-type";
+import { Span } from "./span";
+
+const meta: WsComponentMeta<typeof Span> = {
+  Icon: BrushIcon,
+  Component: Span,
+  canAcceptChildren: false,
+  isContentEditable: false,
+  label: "Span Text",
```

Comment:
Label is for semantic meaning of that component, Bold Text and Italic Text make sense, this case I would call it "Styled Text" I guess